src/natcap/invest/wind_energy.py

```diff
@@ -1769,7 +1763,7 @@ def _read_csv_wind_data(wind_data_path, hub_height):
             to dictionaries that hold wind data at that location.
 
     """
-    wind_point_df = pandas.read_csv(wind_data_path)
+    wind_point_df = utils.read_csv_to_dataframe(wind_data_path, to_lower=False)
```

Comment:
Here's another place where it would be nice to have everything in lowercase, but all-uppercase columns are used throughout this model. I'd like to standardize it (to either upper- or lower-case) for all the models, but it seems like there might be a preference for uppercase here?

Reply:
You know, the all-uppercase thing might just be an artifact of age.  Both HRA and Wind Energy were created back when our standard table input format was `.dbf`, and I believe that ArcGIS would create column names that were in all-caps.

I personally don't have a preference about all-uppercase vs. all-lowercase, so long as we can support case-insensitivity.  In this particular case @dcdenu4 implemented the model.  Doug, do you remember if we made a conscious choice in Wind Energy about using all-uppercase or all-lowercase column indexing?

Reply:
That was indeed a long time ago. Off the top of my head I have no recollection of an all-uppercase vs. all-lowercase decision.  I'd have to check it out.

Reply:
Okay, so I think the Wind model is probably okay for moving to lowercase, though some constants in the model are hard coded to uppercase. But there is a CSV input that is constructed such that the first column is a list of variable names and the second column are the values. 

Wave Energy also has some non standard CSV inputs and usage of shapefiles for column access. I think you're right James, I think the dbf files upper case the column names. Oh, Wave Energy also has a CSV input of just numbers. It's a wave machine performance input. 

So there are certainly some exceptions out there! But they might not play a role in this PR.